rotkehlchen/tests/unit/test_data_updates.py

```diff
@@ -208,19 +282,64 @@ def test_no_update_performed(data_updater: RotkiDataUpdater) -> None:
                 (UpdateType.GLOBAL_ADDRESSBOOK.serialize(), 999),
             ],
         )
+    with ExitStack() as stack:
+        stack.enter_context(patch('requests.get', wraps=make_mock_github_response(latest=1)))
+        patches = [
+            stack.enter_context(patch.object(data_updater, f'update_{update_type.value}'))
+            for update_type in UpdateType
+        ]
+        reset_update_type_mappings(data_updater)
+        data_updater.check_for_updates()
+
+    assert all(patch.call_count == 0 for patch in patches)
+    with data_updater.user_db.conn.read_ctx() as cursor:
+        cursor.execute(  # also make sure latest DB value is not changed
+            'SELECT value from settings WHERE name IN(?, ?, ?, ?)',
+            [x.serialize() for x in UpdateType],
+        )
+        assert {x[0] for x in cursor} == {'999'}
+
+
+@pytest.mark.parametrize('our_version', ['1.35.0'])  # set a normal version for CI
```

Comment:
putting this here only for CI, since in local we do `pip install -e .` and have a proper version but in CI we don't